pages/common/gitleaks.md

```diff
@@ -0,0 +1,32 @@
+# gitleaks
+
+> A scanner for detecting secrets and API keys leaked in Git repositories.
```

Comment:
```suggestion
> Detect secrets and API keys leaked in Git repositories.
```
The previous description was a valid one, but we generally start with a verb in the imperative mood.

Or:

```suggestion
> Scan for secrets and API keys leaked in Git repositories.
```